client/src/components/common/CardWithImage.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Col } from 'reactstrap';
+import PropTypes from 'prop-types';
+import InfoCard from './InfoCard';
+import Image from './Image';
+
+const CardWithImage = (props) => {
```

Comment:
You can destructure the props here only if you are not passing the entire/ spreading props in any other component.

Reply:
Cool. Will keep this in mind from now on. Thanks!